deployment/cdk/opensearch-service-migration/lib/service-stacks/opensearch-container-stack.ts

```diff
@@ -0,0 +1,69 @@
+import {StackPropsExt} from "../stack-composer";
+import {IVpc, SecurityGroup} from "aws-cdk-lib/aws-ec2";
+import {PortMapping, Protocol, Ulimit, UlimitName} from "aws-cdk-lib/aws-ecs";
+import {Construct} from "constructs";
+import {MigrationServiceCore} from "./migration-service-core";
+import {StringParameter} from "aws-cdk-lib/aws-ssm";
+import {ServiceConnectService} from "aws-cdk-lib/aws-ecs/lib/base/base-service";
+
+export interface OpenSearchContainerProps extends StackPropsExt {
+    readonly vpc: IVpc
+}
+
+/**
+ * This is a testing stack which deploys a SINGLE node OpenSearch cluster as the target cluster for this solution with
+ * no persistent storage. As such, it should NOT be used for production use cases.
+ */
+export class OpenSearchContainerStack extends MigrationServiceCore {
+
+    constructor(scope: Construct, id: string, props: OpenSearchContainerProps) {
+        super(scope, id, props)
+        let securityGroups = [
+            SecurityGroup.fromSecurityGroupId(this, "serviceConnectSG", StringParameter.valueForStringParameter(this, `/migration/${props.stage}/${props.defaultDeployId}/serviceConnectSecurityGroupId`)),
+        ]
+
+        const servicePort: PortMapping = {
+            name: "opensearch-connect",
+            hostPort: 9200,
+            containerPort: 9200,
+            protocol: Protocol.TCP
+        }
+        const serviceConnectService: ServiceConnectService = {
+            portMappingName: "opensearch-connect",
+            dnsName: "opensearch",
+            port: 9200
+        }
+        const ulimits: Ulimit[] = [
+            {
+                name: UlimitName.MEMLOCK,
+                softLimit: -1,
+                hardLimit: -1
+            },
+            {
+                name: UlimitName.NOFILE,
+                softLimit: 65536,
+                hardLimit: 65536
+            }
+        ]
+
+        this.createService({
+            serviceName: "opensearch",
+            dockerImageRegistryName: "opensearchproject/opensearch:2",
+            securityGroups: securityGroups,
+            environment: {
+                "cluster.name": "os-docker-cluster",
+                "node.name": "opensearch-node1",
```

Comment:
should you say 'test' or target' or target-test?

Reply:
Just to be clear is this in reference to the `node.name`? In which case I don't have a huge preference would like to have the suffix `-node` on it